specification/ai/DocumentIntelligence/models.tsp

```diff
@@ -520,10 +512,9 @@ model AnalyzeBatchResult {
   skippedCount: int32;
 
   @doc("Operation detail for each document in the batch.")
-  details: AnalyzeBatchOperationDetail[];
+  details?: AnalyzeBatchOperationDetail[];
```

Comment:
Wondering why it becomes an optional property.

Usually, an empty array would just be `[]` but not absent -- unless this is used as intermittent polling response where we want to say `details` not ready.

Reply:
This is due to a missing visibility option in TypeSpec Swagger emitter to allow `details` to be omitted from LIST operation.  I filed a feature request on this, but haven't hear anything back.